laws/traversable.js

```diff
@@ -8,33 +8,35 @@ const {of, traverse, map} = require('..');
 
 ### Traversable
 
-1. `t(u.traverse(x => x, F.of))` is equivalent to `u.traverse(t, G.of)`
-for any `t` such that `t(a).map(f)` is equivalent to `t(a.map(f))` (naturality)
+1. `t(u.traverse(F, x => x))` is equivalent to `u.traverse(G, t)` for any
+   `t` such that `t(a).map(f)` is equivalent to `t(a.map(f))` (naturality)
 
-2. `u.traverse(F.of, F.of)` is equivalent to `F.of(u)` for any Applicative `F` (identity)
+2. `u.traverse(F, F.of)` is equivalent to `F.of(u)` for any Applicative `F`
+   (identity)
 
-3. `u.traverse(x => new Compose(x), Compose.of)` is equivalent to
-   `new Compose(u.traverse(x => x, F.of).map(x => x.traverse(x => x, G.of)))` for `Compose` defined below and any Applicatives `F` and `G` (composition)
+3. `u.traverse(Compose, x => new Compose(x))` is equivalent to
+   `new Compose(u.traverse(F, x => x).map(x => x.traverse(G, x => x)))` for
+   `Compose` defined below and any Applicatives `F` and `G` (composition)
 
 **/
 
-const naturality = t => eq => x => {
-  const a = identity(t(x)[traverse](y => y, t[of]));
-  const b = t(x)[traverse](identity, t[of]);
+const naturality = T => t => eq => x => {
+  const a = identity(t(x)[traverse](T, y => y));
+  const b = t(x)[traverse](T, identity);
   return eq(a, b);
 };
 
-const identityʹ = t => eq => x => {
+const identityʹ = T => t => eq => x => {
   const u = [x];
 
-  const a = u[traverse](Id[of], Id[of]);
```

Comment:
I think in this old version, it should have been:
```js
const a = u[traverse](Id[of], Id[of]);
const b = t(u);
```

and because of that, new version should be like this:
```js
const identityʹ = T => eq => x => {
  const u = [x];
  const a = u[traverse](T[of], T);
  const b = T[of](u);
  return eq(a, b);
}
```

and on caller side:
```js
test(traversable.identity(Id)(equality))
```